x-pack/plugins/alerting/server/task_runner/create_execution_handler.ts

```diff
@@ -163,6 +124,10 @@ export function createExecutionHandler<
     let ephemeralActionsToSchedule = maxEphemeralActionsPerRule;
 
     for (const action of actions) {
+      if (alertExecutionStore.numberOfTriggeredActions >= formattedExecutionConfig.actions.max) {
```

Comment:
This brings up the question of whether we should be keeping track of the number of scheduled actions if the number of triggered actions is capped at the config max. I think we might need to keep track of both for telemetry purposes to see how often we're scheduling above the max allows WDYT?

Reply:
Very good idea, we should definitely keep the both numbers.
But this sounds like new event log and telemetry object, should we do it within this issue?
And i think the new number's name should be numberOfActionsToTrigger :)

Reply:
Created a follow up issue for it:
https://github.com/elastic/kibana/issues/127448